Alignment/CommonAlignmentProducer/plugins/GlobalTrackerMuonAlignment.cc

```diff
@@ -3417,11 +3417,11 @@ void GlobalTrackerMuonAlignment::writeGlPosRcd(CLHEP::HepVector& paramVec) {
             << std::endl;
   std::cout << calo.rotation() << std::endl;
```

Comment:
Not really for this PR, but these `cout`-s should be changed.

Reply:
I am not the maintainer of this code.